### ISSUE TITLE
skhd -o doesn't print keycodes

### ISSUE THREAD

**Issue by alptugan** [koekeishiya/skhd#309]
When I type `skhd -o` Terminal return the char pressed on the keyboard as it is. E.g. If you hit 'c', it return 'c' again instead of 0x... something.
My setup: latest Ventura OS, latest brew, Macbook M2

**Comment by tjex** [comment]
...wow. Thats super odd. I'm on close enough of the same system to you and it works for me. 

Can you post a screen shot(s)?

**Comment by alptugan** [comment]
Here they are. It continuously prints the key in latin.
![Screenshot 2023-12-02 at 20 13 23](https://github.com/koekeishiya/skhd/assets/315287/6f38468b-9480-4e81-b47c-347471da1de2)
![Screenshot 2023-12-02 at 20 12 42](https://github.com/koekeishiya/skhd/assets/315287/be51bdab-3e50-43ec-b684-4043eba30873)

**Comment by alptugan** [comment]
Yes! exactly it is :)
I've given all permissions and been using with Yabai for months. It's functioning as expected, btw. I can use my custom scripts and automation stuff using shortcuts via skhd. But the `skhd -o` it stopped working suddenly. 

`...out.log` is empty and `...err.log` says "could not locate the window to act on!" I guess it is related to Yabai.

I have tried `skhd --stop-service` & `skhd --start-service`. No success.

**Comment by tjex** [comment]
Hmm, would yabai output to skhd log?

Can you post your skhdrc?

**Comment by alptugan** [comment]
Last night I executed `brew update; brew upgrade`. I was going to `cat` the `skhdrc` to post here. I've checked `skhd -o` one last time whether it works or not! Well, good news. It is working now. 
![Screenshot 2023-12-03 at 15 25 44](https://github.com/koekeishiya/skhd/assets/315287/c26912f6-33ee-4d37-898f-c0fb31d18378)

I don't know why, but after the `brew` stuff Yabai and skhd asked for permission. I have two Yabai app seems running. But everything is working as expected right now. 
![Screenshot 2023-12-03 at 15 29 43](https://github.com/koekeishiya/skhd/assets/315287/5da0f4b6-3ef8-4d47-992a-8b97ecfb4e88)

In short, I guess updating brew, restart computer and enable-disable skhd steps fix the issue. Thanks for your support @tjex, btw
